tests/test_redmine.py

```diff
@@ -0,0 +1,461 @@
+import logging
+from collections import namedtuple
+from unittest import main, TestCase
+from unittest.mock import patch
+
+import requests
+import ticketutil.redmine as Redmine
```

Comment:
Want to change this to the following?

`import ticketutil.redmine as redmine`

Just to stay consistent with all the other unit test files you've written, which use a lowercase name for the tool.